### DISCUSSION TITLE
CodeQL score table removed from documentation?

### DISCUSSION THREAD

**Discussion by Nickog13** [github/securitylab#236] [Q&A] [answered]
Hey guys,

I have a question concerning the codeql documentation.
I checked on the documentation a few weeks ago and saw a table regarding how the codeql built-in queries perform against the NIST Juliet testsuite (I think the table showed precision, recall values, etc.).
However, I could not find that specific table this morning (I actually wanted to use it for a presentation).
Has it been removed from the docs? Or moved to some other place?

Thanks in advance!

Greetings
Dominik

**Comment by criemen** [comment, marked as answer]
Hi Dominik,

please excuse the late reply.
You are correct, we indeed used to publish SAMATE test results.
When moving to our new docs site, that was lost.
I filed a ticket internally, and we will be working on getting the SAMATE results back into our documentation.
However, it might take a bit of time until we get there, please bear with us until then!
